index.js

```diff
@@ -87,7 +87,14 @@ function Log(options) {
 
   if (options.rawTransports){
     logTransports = options.transports
-  } else {
+  }
+  else if(options.transports.length == 0){
+    //defaults to console if no transports are given
```

Comment:
Seems reasonable. I wonder if, after upgrading, this will cause the test suites in neodarwin, etc, to start emitting lots of log output. If so, then we could address by setting a console transport in the test configs for each repo, and using the `silent: true` option to suppress output by default. (Sometimes, I've *wanted* to see log output when working on a test, and I used to go into my local config overrides and add a console transport to be able to, so this would make it easier, b/c I could just switch to `silent: false` temporarily.)

Reply:
Good point! Just tested in neodarwin & it doesn't look like it causes extra console logs.

Reply:
But it does look like it is causing extra logs in sd-examples. When upgrading we could add the silent flag to the repo as you suggested.